Antonio_Ramos_Documentation

```diff
@@ -0,0 +1,171 @@
+"
+
+- found file "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp" however when I tried to open it with python code I used earlier, I got the following error message
+"
+DriverError: G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp: No such file or directory
+"
+
+- noticed that within the file "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data" there are updated versions of the "GoodLabs.shp" and "GoodLabs.dbf" named "GoodLabs_New.shp" and "GoodLabs_New.dbf" will update the file names within the K-function Local_Scipy.py script to the new names.
+
+- rewrote the path name on line 39 to account for the updated GoodLabs.
+old od - "pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp""
+new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs_New.shp""
+
+- ipython console displays the run file command but then doesnt return anything and seems to freeze since it doesnt let me run anyother scripts, will attempt to restart and run again
+
+- again no response within the console and unabe to run other scripts
+
+- reverted path name to original - " pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Base Data\GoodLabs.shp""
+and got shapefile exception again when trying to run the script 
+
+- realized I was working on the wrong script and started working on the K-function Local_3Stage.py located inG:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions
+
+- once I started working on the correct script I realized that the Will's dosumentation is not up to date and does not reference the same files as the script does. So, I created an excell spreadsheet to list all the files mentioned by Will in his documentation in an attemtp to better understand where everything was coming from, what it did, and how it should function with the python scripts. 
+
+- emailed Will asking if he had any more updated documentation. Will responded with the most comprehensive documentation that he has, however it still does not mention the files being refrenced by the K-function Local_3Stage.py script. 
+
+- I will now begin doing a line by line analysis of the K-function Local_3Stage.py script to better understand what it is doing with the data files being referenced 
+
+- All modules necessary for running K-function Local_3Stage.py installed onto computer to  C:\Apps-SU\Anaconda3\pkgs. Able to fully run trial version of 3 stage script, K-function Local_3Stage Trial.py, however when proceeding to try and run the K-function Local_3Stage.py script all modules are imported and programs runs effectively until it stars performing an endless loop of simulations at line 125. Printing the following over and over again "SIMULATION #### Threw and error." with "####" being respective simulation numbers.
```

Comment:
Does it print this error for every single simulation number, or just some of them?

Reply:
It just goes into an infinite loop where it prints that for every simulation and never stops. So it will just keep printing, "SIMULATION 1 Threw an error" then "SIMULATION 2 Threw an error" ad infinitum. If I don't stop it, it will just keep going.

Reply:
Good to know. The reason I was asking is that if it was only for some simulation numbers and not others, it might indicate a different kind of problem. Please clarify in your notes that it's for every value of the simulation variable (I didn't know how to interpret "respective")

Reply:
This is the one outstanding issue on this pull request. Please fix it on your next update so I can close this pull request. It gets confusing if too many are open at the same time.

Reply:
my apologies, I thought I had fixed it. I will update it in the next pull request.